common/src/main/java/eu/europeana/edm/ocr/AnnotationsGenerator.java

```diff
@@ -0,0 +1,159 @@
+package eu.europeana.edm.ocr;
+
+import eu.europeana.edm.FullTextAnnotation;
+import eu.europeana.edm.FullTextPackage;
+import eu.europeana.edm.media.MediaReference;
+import eu.europeana.edm.text.FullTextResource;
+import eu.europeana.edm.text.TextBoundary;
+import eu.europeana.edm.text.TextReference;
+import eu.europeana.fulltext.AnnotationType;
+import eu.europeana.fulltext.alto.model.*;
+
+import static eu.europeana.fulltext.alto.parser.EDMFullTextUtils.newTextBoundary;
+
+/**
+ * @author Hugo Manguinhas <hugo.manguinhas@europeana.eu>
+ * @since 22 Jun 2018
+ */
+public class AnnotationsGenerator extends AbsAltoVisitor {
+
+    private StringBuilder sb = new StringBuilder(100 * 1024);
+    private FullTextPackage page;
+
+
+    /**
+     * Process the alto page to Fulltext package
+     * @param altoPage
+     * @param ref
+     * @return
+     */
+    public synchronized FullTextPackage process(AltoPage altoPage, MediaReference ref) {
```

Comment:
Have removed synchronized. Now that I have made a few changes I see there is no need for a local sb variable. When I tested it everything works alright. I see the proper value of Resource in the string. But if it's otherwise let me know